docker/domserver/configure.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -e
```

Comment:
Is it possible to also use `/bin/sh` here, as well as in the scripts below?

If there's a really good reason not to, that's ok, but I think it's better to try and use posix shell.

Reply:
I think it should work, I just used bash by habit. I'll have a look later. Thanks for you feedback.

Reply:
It works everywhere except in the domjudge initialization which requires indirect substitution for the `file_or_env` thing